README.md

```diff
@@ -22,13 +22,17 @@ At this step, the upgrader checks the project against a set of rules. The main g
 
 To evaluate a project, execute the following command from its directory:
 ```bash
-bin/console analyze:php:code-compliance --format=yaml
+bin/console analyze:php:code-compliance
 ```
+The command generates a YAML report at `{project_directory}/reports/`.
+
+Available options:
+- Module filtration option, use it for specifies which modules should be evaluated.
```

Comment:
I think it is better to write:

-m - module filtration option. It is used to specify the modules for evaluation.